shared/oae/bundles/ui/default.properties

```diff
@@ -587,6 +587,7 @@ TEXT_DOCUMENT = Text document
 THE_PASSWORD_SHOULD_BE_AT_LEAST_SIX_CHARACTERS_LONG = The password should be at least 6 characters long
 THE_USERNAME_SHOULDNT_CONTAIN_SPACES = The username shouldn't contain spaces
 THE_USERNAME_SHOULD_BE_AT_LEAST_THREE_CHARACTERS_LONG = The username should be at least 3 characters long
+THIS_EMAIL_DOMAIN_IS_NOT_ALLOWED = This email domain is not allowed
```

Comment:
Something along the lines of `You can only register with an 'ox.ac.uk' email address` would be friendlier